ingestion/src/metadata/ingestion/source/messaging/kinesis/metadata.py

```diff
@@ -57,48 +66,60 @@ def create(cls, config_dict, metadata_config: OpenMetadataConnection):
             )
         return cls(config, metadata_config)
 
-    def get_topic_list(self) -> Iterable[BrokerTopicDetails]:
-        all_topics, has_more_topics, args = [], True, {"Limit": 100}
-        try:
-            while has_more_topics:
+    def get_stream_names_list(self) -> List[str]:
+        """
+        Get the list of all the streams
+        """
+        all_topics, has_more_topics, args = [], True, {KinesisEnum.LIMIT.value: 100}
```

Comment:
I still believe args can be an object called `KinesisArgs` that is created like this:
```python
KinesisArgs(limit=100)
```
Then when you have to use it to call `list_streams`, you can do `**args.dict()`. wdyt?

Reply:
Ok I'll do that